test/testlib.sh

```diff
@@ -525,7 +525,12 @@ enable_actions() {
 }
 
 is_actions_enabled() {
-  ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'
+  # ghe-ssh will always return exit code 0.  Evaluate property value instead.
+  if $( ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config app.actions.enabled' ) = "true"; then
```

Comment:
'ghe-config app.actions.enabled' is possible to return non zero exit code if this is not set. Probably a combination of `--present` and `--true` would help here

Reply:
there are hackers ising my info vell numbers and many account also be anle to change passwords  using Otiplex